package.json

```diff
@@ -36,10 +36,12 @@
     "babel-preset-stage-1": "^6.5.0",
     "babel-register": "^6.11.6",
     "babel-runtime": "^6.11.6",
+    "bootstrap": "^3.3.7",
```

Comment:
There're lots of code are based on jquery and bootstrap. The jquery and bootstrap are imported in index.html by following two lines of code.
```html
<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```
But in the unit test, these two libs has to be imported by node lib.